Dockerfile-localnet

```diff
@@ -19,6 +19,8 @@ COPY go.sum .
 RUN go mod download
 COPY version.sh .
 COPY --exclude=*.sh --exclude=*.md --exclude=*.yml . .
+ARG NODE_VERSION
+ARG NODE_COMMIT
```

Comment:
nit: is there are plan to use these vars inside Dockerfile?

Reply:
Yeah updated